src/core/environment.js

```diff
@@ -199,7 +199,7 @@ p5.prototype.focused = document.hasFocus();
  * strings: `'help'`, `'wait'`, `'crosshair'`, `'not-allowed'`, `'zoom-in'`,
  * and `'grab'`. If the path to an image is passed, as in
  * `cursor('assets/target.png')`, then the image will be used as the cursor.
- * Images must be in .cur, .gif, .jpg, .jpeg, or .png format.
+ * Images must be in .cur, .gif, .jpg, .jpeg, or .png format and should be of 32 by 32 pixels dimensions.
```

Comment:
Thanks for finding this limit, I wasn't aware of it! I found a source for the limit on MDN, how do you feel about linking to it?

```suggestion
 * Images must be in .cur, .gif, .jpg, .jpeg, or .png format and should be <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#icon_size_limits">at most 32 by 32 pixels large.</a>
```

Reply:
Yes we can link to it.

Reply:
Hey, I think we should merge it, so users donot get confused. Thanks